subprojects/packagefiles/libgpiod/bindings/cxx/tests/meson.build

```diff
@@ -0,0 +1,21 @@
+# SPDX-License-Identifier: LGPL-2.1-or-later
+# Copyright 2021 IBM Corp.
+
+catch2_dep = dependency('catch2')
```

Comment:
Randomly noticed this:

```
FAILED: subprojects/libgpiod-1.6.3/bindings/cxx/tests/gpiod-cxx-test.p/gpiod-cxx-test-main.cpp.o 
ccache c++ -Isubprojects/libgpiod-1.6.3/bindings/cxx/tests/gpiod-cxx-test.p -Isubprojects/libgpiod-1.6.3/bindings/cxx/tests -I../subprojects/libgpiod-1.6.3/bindings/cxx/tests -Isubprojects/libgpiod-1.6.3/bindings/cxx -I../subprojects/libgpiod-1.6.3/bindings/cxx -Isubprojects/libgpiod-1.6.3/include -I../subprojects/libgpiod-1.6.3/include -Isubprojects/libgpiod-1.6.3/tests/mockup -I../subprojects/libgpiod-1.6.3/tests/mockup -I../subprojects/Catch2-2.13.3/single_include -fdiagnostics-color=always -D_FILE_OFFSET_BITS=64 -Wall -Winvalid-pch -Wnon-virtual-dtor -Wextra -g -pthread -MD -MQ subprojects/libgpiod-1.6.3/bindings/cxx/tests/gpiod-cxx-test.p/gpiod-cxx-test-main.cpp.o -MF subprojects/libgpiod-1.6.3/bindings/cxx/tests/gpiod-cxx-test.p/gpiod-cxx-test-main.cpp.o.d -o subprojects/libgpiod-1.6.3/bindings/cxx/tests/gpiod-cxx-test.p/gpiod-cxx-test-main.cpp.o -c ../subprojects/libgpiod-1.6.3/bindings/cxx/tests/gpiod-cxx-test-main.cpp
In file included from /usr/include/signal.h:328,
                 from ../subprojects/Catch2-2.13.3/single_include/catch2/catch.hpp:8034,
                 from ../subprojects/libgpiod-1.6.3/bindings/cxx/tests/gpiod-cxx-test-main.cpp:9:
../subprojects/Catch2-2.13.3/single_include/catch2/catch.hpp:10822:58: error: call to non-‘constexpr’ function ‘long int sysconf(int)’
10822 |     static constexpr std::size_t sigStackSize = 32768 >= MINSIGSTKSZ ? 32768 : MINSIGSTKSZ;
      |                                                          ^~~~~~~~~~~
In file included from /usr/include/bits/sigstksz.h:24,
                 from /usr/include/signal.h:328,
                 from ../subprojects/Catch2-2.13.3/single_include/catch2/catch.hpp:8034,
                 from ../subprojects/libgpiod-1.6.3/bindings/cxx/tests/gpiod-cxx-test-main.cpp:9:
/usr/include/unistd.h:641:17: note: ‘long int sysconf(int)’ declared here
  641 | extern long int sysconf (int __name) __THROW;
      |                 ^~~~~~~
In file included from ../subprojects/libgpiod-1.6.3/bindings/cxx/tests/gpiod-cxx-test-main.cpp:9:
../subprojects/Catch2-2.13.3/single_include/catch2/catch.hpp:10881:45: error: size of array ‘altStackMem’ is not an integral constant-expression
10881 |     char FatalConditionHandler::altStackMem[sigStackSize] = {};
      |                                             ^~~~~~~~~~~~
```

It works fine if I acquire 2.13.6 from my distro... or if I update catch2.wrap from 2.13.3, weird.

Reply:
Meh, according to google, MINSIGSTKSZ is a glibc change, conveniently my system runs glibc-git.

Reply:
In fact, also if I rebase to grab #132

Reply:
I'll add `version: '>=2.13.7'`

Reply:
I wouldn't. libgpiod doesn't depend on any specific version of catch2, only glibc depends on a specific version of catch2.

(Also, it is fixed in .5, not .7)

Reply:
Using directly via the wrapdb, people will automatically get .7, but using distro dependencies, whichever version comes from the distro should be updated enough to be compatible with the distro glibc, which might be less than .7 or even .5